PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+What was changed in this pull request?
+
+Why is it necessary?
+
+## Checklist
+
+[ ] Unit tests added or updated
```

Comment:
Shall we turn this into a [checkbox list](https://help.github.com/articles/about-task-lists/) using?
```
- [ ] Unit...
```
E.g.:
- [ ] Unit...

Reply:
Yes, that's what I was going for but didn't quite accomplish.